### PR TITLE
README

### PR DESCRIPTION
**Pull request by vadymmarkov**

### CODE REVIEW

**Comment by onmyway133** [comment]
@vadymmarkov do we have any struct or class named `BarcodeScanner` ?

**Comment by vadymmarkov** [comment]
@onmyway133 Nope 😄

**Comment by zenangst** [comment]
I like this PR

**Comment by vadymmarkov** [comment]
@zenangst Let's merge it then 😄 Have you read readme, is everything fine there?

**Comment by zenangst** [comment]
giphy-75

File: README.md [hunk 1]
```diff
@@ -1,24 +1,189 @@
...
+
+### Customization
+
+We styled **BarcodeScanner** to make it look nice, but feel free to customize
+its appearance by changing global configuration variables:
+
+```swift
+// Strings
+BarcodeScanner.Title.text = NSLocalizedString("Scan barcode", comment: "")
+BarcodeScanner.CloseButton.text = NSLocalizedString("Close", comment: "")
```
Comment (onmyway133) [inline README.md]: Should these be on some kind of `Config` ?

Comment (vadymmarkov) [inline README.md]: @onmyway133 I don't know, if feel it's kinda unnecessary.

**Pull request merged**